SentinelOne/sentinelone/tests/activity-type-5232.json

```diff
@@ -35,7 +39,12 @@
       "id": "551799238352448315"
     },
     "rule": {
-      "id": "556166862007673241"
+      "description": "Flux",
+      "id": "556166862007673241",
+      "name": "Block all"
```

Comment:
Strange, no ?

Reply:
What is strange ?

It comes from the following fields:
```json
    "ruleDescription": "Flux",
    "ruleId": 556166862007673241,
    "ruleName": "Block all",
```

Reply:
Fair enough, I just find strange to have only "Flux" in description and "Block All" as the rulename

Reply:
:shrug: